app/services/katello/pulp3/repository/docker.rb

```diff
@@ -0,0 +1,49 @@
+require 'pulp_docker_client'
+
+module Katello
+  module Pulp3
+    class Repository
+      class Docker < ::Katello::Pulp3::Repository
+        def api_client
+          PulpDockerClient::ApiClient.new(smart_proxy.pulp3_configuration(PulpDockerClient::Configuration))
+        end
+
+        def remote_class
+          PulpDockerClient::DockerRemote
+        end
+
+        def remotes_api
+          PulpDockerClient::RemotesDockerApi.new(api_client)
+        end
+
+        def distribution_class
+          PulpDockerClient::DockerDistribution
+        end
+
+        def distributions_api
+          PulpDockerClient::DistributionsDockerApi.new(api_client)
+        end
+
+        def remote_options
+          if root.url.blank?
+            super
+          else
+            if root.docker_tags_whitelist.any?
+              common_remote_options.merge(url: root.url, upstream_name: root.docker_upstream_name, whitelist_tags: root.docker_tags_whitelist.join(","))
+            else
+              common_remote_options.merge(url: root.url, upstream_name: root.docker_upstream_name, whitelist_tags: nil)
+            end
```

Comment:
could probably simplify this a bit to not repeat the full line

Reply:
Fixed this in my latest commit.